packages/unlock-express/index.js

```diff
@@ -0,0 +1,167 @@
+const ethers = require('ethers')
+const keyExpirationFor = require('./src/keyExpirationFor')
+/**
+ * Main function that yields the middleware
+ * @param {*} config
+ * @param {*} app
+ * @returns
+ */
+const configureUnlock = (config, app) => {
+  if (!config.baseAuthRedirectUri) {
+    // eslint-disable-next-line no-param-reassign
+    config.baseAuthRedirectUri = '/unlock-callback'
+  }
+
+  if (!config.providers) {
+    // eslint-disable-next-line no-param-reassign
+    config.providers = {}
+  }
+
+  if (!config.providers[1]) {
+    // eslint-disable-next-line no-param-reassign
+    config.providers[1] =
+      'https://mainnet.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161'
```

Comment:
Are you sure you want to use your private infura keys here? If they exceed their limits, the calls will stop working and people will not be able to figure out why

Reply:
We need to put a default config tho, don't we?